test/core/TakerOperations.t.sol

```diff
@@ -1152,6 +1152,21 @@ contract TakerOperationsTest is MangroveTest {
     Leaf emptyLeaf = leaf.setTickFirst(tick, 0).setTickLast(tick, 0);
     assertTrue(emptyLeaf.isEmpty(), "leaf should not have other tick used");
   }
+
+  /* 
+  An attempt to check for overflow when accumulating sor.gives into mor.totalGave.
+  I have not found a way to actually trigger it by mutating state somewhere.
+  This test just considers as many offers as possible that each have a maximal `wants` and makes sure the error will be about stack overflow, not uint overflow. 
+  */
+  function test_maximal_wants_is_ok() public {
+    uint maxOfferWants = LogPriceLib.inboundFromOutboundUp(MAX_LOG_PRICE, type(uint96).max);
```

Comment:
isn't this only covering the fillWants=true scenario? which actually sets sor.gives and not sor.wants?

Reply:
Ah, when fillWants=false then fillVolume will be the max we give, and it can obviously not be above uint256.max?